config/default/crds/serving_v1alpha1_kfservice.yaml

```diff
@@ -6,10 +6,25 @@ metadata:
     controller-tools.k8s.io: "1.0"
   name: kfservices.serving.kubeflow.org
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.uri.external
+    name: uri
+    type: string
+  - JSONPath: .status.conditions[?(@.type==\"READY\")].status
+    name: status
+    type: string
+  - JSONPath: .status.default.traffic
+    name: defaulttraffic
+    type: integer
+  - JSONPath: .status.canary.traffic
+    name: canarytraffic
+    type: integer
```

Comment:
how about adding another one as below:
```
  - JSONPath: .metadata.creationTimestamp
    name: Age
    type: date
```

Reply:
This is included by default, I think.

Reply:
AFAIK, `Age` is included automatically only when no user-defined additionalPrinterColumns is specified. In case here, I don't think it will be added.
Maybe you can take a test

Reply:
Nice -- will update. "Take a test"? "Make a test?"

Reply:
I mean you can verify if Age is included automatically here  ：)